deps/clew/src/clew.cpp

```diff
@@ -164,7 +164,7 @@ int clewInit()
 #ifdef _WIN32
     const char *paths[] = {"OpenCL.dll", NULL};
 #elif defined(__APPLE__)
-    const char *paths[] = {"/Library/Frameworks/OpenCL.framework/OpenCL", NULL};
+    const char *paths[] = {"/System/Library/Frameworks/OpenCL.framework/OpenCL", NULL};
```

Comment:
This patch is a bad idea, it should be:

```
const char *paths[] = {"/Library/Frameworks/OpenCL.framework/OpenCL", 
 "/System/Library/Frameworks/OpenCL.framework/OpenCL", NULL};
```

If you want, add a paths to the list but don't remove them.

Reply:
/Library/Frameworks/OpenCL does not even exist !!!

/System/Library/Frameworks/OpenCL  is what we need for standard.

/Library/frameworks is where the user ( installed ) Frameworks go.. Cuda etc

OpenCL is a System Framework. ... as far as i know

Reply:
I think if Xcode is installed the sdk Frameworks are used ... if not the one in /System/Library/Framewoks

Reply:
"/Library/Frameworks/OpenCL.framework/OpenCL" is the path used by Blender so it should be valid (in some case).

Reply:
Hmm OpenCL is disabled on OSX  for long. So maybe that is deprecated..
on Catalina OpenCL is also in /System/Library 
Luxcore won't run on everything less that 10.13 
But if you want we just add it ..

Reply:
I added it 
[https://github.com/LuxCoreRender/LuxCore/pull/388](url)

Reply:
I added it back
[https://github.com/LuxCoreRender/LuxCore/pull/388](url)

Reply:
Umm, this broken link!